src/themes/ionic.mixins.scss

```diff
@@ -157,16 +153,44 @@
   }
 }
 
-$include-rtl: true !default;
+$app-direction: null !default;
+
+@mixin multi-dir() {
+  @if $app-direction == null {
+    $root: #{&};
+    @at-root [dir="ltr"], [dir="rtl"] {
```

Comment:
Because of CSS specificity rules, otherwise it overrides stuff. And no, the `@at-root` seems to be a must